content/en/mobile_testing/mobile_app_tests/steps.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Mobile App Test Steps
+kind: documentation
+description: Learn how to automatically record and manually set steps in a mobile test recording.
+further_reading:
+- link: "/mobile_testing/mobile_app_tests/"
+  tag: "Documentation"
+  text: "Learn about Synthetic mobile tests"
+- link: "/mobile_testing/mobile_app_tests/advanced_options"
+  tag: "Documentation"
+  text: "Learn about advanced options in mobile tests"
+---
+
+## Overview
+
+Steps represent individually recorded interactions or assertions that you want to execute in your test. To define a step, click **Start Recording** and interact with the device as you would normally, or create a step manually by clicking **Assertions** or **Special Actions**. 
+
+## Launch a device
+
+To start recording and adding steps, select a device to launch a mobile app test from the dropdown menu and click **Launch Device**. 
+
+{{< img src="mobile_testing/launch_device.png" alt="Select a device to run a mobile test on" style="width:60%" >}}
+
+Select **Show only available devices. Available devices load faster** to see the most available devices for shorter testing wait times. 
+
+### Notifications
+
+Click the **Device Connection Notification** button to enable notifications for when your device is ready and when your device is going to timeout because of inactivity.
```

Comment:
Let's specify where this is so:

Click the **Device Connection Notification** button in the **Launch a device to start recording** modal to enable notifications for when your device is ready and when your device is going to timeout because of inactivity.

Reply:
```suggestion
Click the **Device Connection Notification** button in the **Launch a device to start recording** modal to enable notifications for when your device is ready and when your device is going to timeout because of inactivity.
```